### PR TITLE
Feature/gear estop

### PR DESCRIPTION
**Pull request by OliviaRS**
**First implementation of eStop**

### CODE REVIEW

**Comment by duane701** [comment]
I resolved the conflicts on this branch from the recent master merges.

File: src/Commands/SetGearRoller.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public:
+	SetGearRoller(double rollerSpeedRPM);
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+
+private:
+	double rollerSpeedRPM;
```
Comment (SkylerPIlot) [inline src/Commands/SetGearRoller.h:16]: wouldn't it be better to write this as `double mRollerSpeedRPM;`? If not both your constructor variable and class variable have the same name.

File: src/RobotMap.h [hunk 2]
```diff
@@ -18,7 +18,7 @@ class RobotMap {
   static const int kIDShooterFeeder = 13;
 
   //..........GearPickup..........
-  static const int kIDRoller = 10;
+  static const int kIDRoller = 3; //10
```
Comment (SkylerPIlot) [inline src/RobotMap.h:21]: This looks like it should be changed back to 10.

File: src/Subsystems/GearPickup.cpp [hunk 3]
```diff
@@ -49,6 +59,23 @@ bool GearPickup::IsGearAlligned() const{
...
+ if (eStop.ShouldStop(RPM, GetGearIntakeRPM())) {
+    CancelCurrentCommand(GetCurrentCommand());
+
+    estophappened = true;
+
+  } else {
+    roller.Set(RPM);
+
+  }
+  SmartDashboard::PutBoolean("ESTOP", estophappened);
```
Comment (SkylerPIlot) [inline src/Subsystems/GearPickup.cpp:76]: We can remove this since we have finished testing.

**Review by SkylerPIlot — COMMENTED** [review]
Just some small style things but other than that LGTM.

**Review by Diego-Andrade — APPROVED** [review]
LGTM

**Pull request merged**